ansi-terminal.cabal

```diff
@@ -1,5 +1,5 @@
 Name:                ansi-terminal
-Version:             0.6.2.3
+Version:             0.6.2.4
```

Comment:
Please don't change the package version. I do it in a separate commit when I make the release. Otherwise I get confused about which version is where. Besides, this is going to be a third-digit increment (since we've added new modules).